wbooks/src/app/components/CommentList/index.tsx

```diff
@@ -9,26 +9,23 @@ import styles from './styles';
 
 const CommentList = ({ comments }: CommentsData) => {
   const allCommentsAmount = comments.length;
-  const [initialAmount, setNumber] = useState(2);
+  const [commentsToView, setNumber] = useState(2);
```

Comment:
Let's rename this. We usually use `X, setX` names for `useState`
```suggestion
  const [commentsToView, setCommentsToView] = useState(2);
```